packages/design-system/src/components/Badge/variants/BadgePopover.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.badge-popover__button {
+	align-items: center;
```

Comment:
35 lines with the same css, it means it can be shared with dropdown or do you have a good reason to keep it seprarated ?